mail/mu/Portfile

```diff
@@ -4,39 +4,49 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           elisp 1.0
 
-github.setup        djcb mu 1.0 v
-license             GPL-3
+name                mu
```

Comment:
... and there will be no need to set the name.

Reply:
same as before